package.json

```diff
@@ -19,9 +19,10 @@
     "eslint-plugin-no-unsanitized": "^3.1.4",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-sonarjs": "0.5.0",
+    "import-sort-style-eslint": "^6.0.0",
```

Comment:
Adding the dev-dependency here did not work for me: you need to add it either to theia/package.json or che-theia/extensions/<che extension>/package.json

Reply:
it is working because all eslint packages are added to theia/package.json

Reply:
and here I just moved a dependency that was in a very deep workspace making it available to all workspaces (should be explicit, not implicit dependency)